scripts/xtensa-build-zephyr.py

```diff
@@ -1169,6 +1170,32 @@ def reproducible_checksum(platform, ri_file):
 	chk256 = sof_ri_info.EraseVariables(ri_file, parsed_ri, west_top / repro_output)
 	print('sha256sum {0}\n{1} {0}'.format(repro_output, chk256))
 
+def create_firmware_bundle():
+	if shutil.which('tar') is None:
+		print("tar tool not found, not creating firmware tarball")
+		return
+
+	# vendor directories (typically under /lib/firmware) to include
+        # in the tarball
+	vendor_dirs = [ 'intel']
+
+	build_top = pathlib.Path(west_top, 'build-sof-staging', 'sof')
+	# special case when build script is called for a single
+	# target only
```

Comment:
Special cases suck. Is this really needed?

Reply:
This is currently needed to work with the build tools we use to prepare sof-bin releases. I added more commentary on how the script is used in this case, and what we are doing to facilitate. I think this is a fairly reasonable compromise.